src/sql/workbench/contrib/notebook/browser/notebookActions.ts

```diff
@@ -448,6 +450,14 @@ export class RunParametersAction extends TooltipFromLabelAction {
 	 * with injected parameters value from the QuickInput
 	*/
 	public override async run(context: URI): Promise<void> {
+		if (context.scheme === Schemas.untitled) {
```

Comment:
Wouldn't it just be easier/cleaner to just hide the run with parameters action for untitled notebooks?

Reply:
Woops, this comment popped up just as I hit the merge button. I'll try that

Reply:
Although, now that I think about it, wouldn't it be worse for discoverability if we hid the button? People starting out with untitled notebooks wouldn't see it

Reply:
It's a discussion we should have - that's always the constant battle we have with this stuff. But in general in ADS we've tended to hide features that aren't supported rather than make it seem like it works and show them an error when they try. 

We could look into just disabling it as well - but then we're limited by the amount of text we can display.

I don't have a huge preference in the end though, @chlafreniere @dzsquared any thoughts/preferences here?